### ISSUE TITLE
'Cookbook' page

### ISSUE THREAD

**Issue by duncanwp** [pydata/xarray#1790]
Would there be interest in adding a 'cookbook' section to the docs a-la [Pandas](http://pandas.pydata.org/pandas-docs/stable/cookbook.html)? The current Recipes section might then be better renamed as Gallery?

It's useful for the kind of thing which isn't a full-fledged example, but might be useful nonetheless. I'll hapily put together a pull-request if there's interest.

**Comment by shoyer** [comment]
Sure, this seems like it could make for a nice addition to the docs!

**Comment by fmaussion** [comment]
> The current Recipes section might then be better renamed as Gallery?

Yes I agree that "Gallery" should be more a collection of plots than something else. The [Parsing rasterio’s geocoordinates](http://xarray.pydata.org/en/latest/auto_gallery/plot_rasterio.html#sphx-glr-auto-gallery-plot-rasterio-py) is something "in between": the actual message is how to parse the coordinates and would be better placed in a cookbook, while the plot is a nice side product and is well placed in the gallery...

**Comment by fmaussion** [comment]
The current [Example](http://xarray.pydata.org/en/latest/examples.html) section is a bit disorganized. Maybe the Cookbook could replace it, and the Gallery could become an independent section?

**Comment by rabernat** [comment]
We will be focusing on this at the SciPy sprint.

**Comment by keewis** [comment]
the work done in the scipy sprints is on master now, with plans for new additions in #3564.

**Comment by kmuehlbauer** [comment]
@keewis Is [xarray-tu](https://tutorial.xarray.dev) what was imagined with the Cookbook page? Should we close this issue and #1791 as well as #3564?

**Comment by keewis** [comment]
not sure. #3564 seems much closer to what https://tutorial.xarray.dev does, while this appears to be more towards a collection of small examples for common tasks (somewhat similar to the `itertools` recipes).

I'd say #1791 is really old so we should be able to close that.
